src/main/java/com/streamr/client/utils/Web3jUtils.java

```diff
@@ -0,0 +1,197 @@
+package com.streamr.client.utils;
+
+import com.streamr.client.dataunion.DataUnionClient;
+import com.streamr.client.dataunion.contracts.IERC20;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.web3j.abi.FunctionEncoder;
+import org.web3j.abi.FunctionReturnDecoder;
+import org.web3j.abi.datatypes.Address;
+import org.web3j.abi.datatypes.DynamicArray;
+import org.web3j.abi.datatypes.Function;
+import org.web3j.abi.datatypes.Type;
+import org.web3j.crypto.Sign;
+import org.web3j.protocol.Web3j;
+import org.web3j.protocol.core.DefaultBlockParameterName;
+import org.web3j.protocol.core.methods.request.Transaction;
+import org.web3j.protocol.core.methods.response.EthCall;
+import org.web3j.protocol.core.methods.response.TransactionReceipt;
+
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.util.*;
+
+public class Web3jUtils {
```

Comment:
This class is not part of DU2 API. Let's define it as package local.

Reply:
Eh I could, but I don't think it's necessary. Web3jUtils contains generic static methods to help with web3j transactions. @hpihkala what do you think?

Reply:
it's not in same package. lets keep the class visible for now